sonic_platform_base/sonic_xcvr/xcvr_api_factory.py

```diff
@@ -36,15 +40,38 @@ def _get_id(self):
             return None
         return id_byte_raw[0]
 
+    def _get_vendor_name(self):
+       name_data = self.reader(129, 16)
+       if name_data is None:
+           return None
+       vendor_name = str(name_data, 'UTF-8')
+       return vendor_name.strip()
+
+    def _get_vendor_part_num(self):
+       part_num = self.reader(148, 16)
```

Comment:
@AnoopKamath could you define 148 and 16 same as for vendor name

Reply:
Taken care